{"name": "Проверка допустимого количества символов в полях ввода", "status": "failed", "statusDetails": {"message": "AssertionError: No message", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4cbde42746b7f3fa3ec935c52c3b5085\")>, base_url = 'http://172.19.16.142/', first_name = 'e39akyNUA4ZfnglSV5sX2bFYQrhTqjIi', second_name = 't'\npassword = 'e$kWq2i+a&cY9bJtB4OI'\n\n    @allure.title(\"Проверка допустимого количества символов в полях ввода\")\n    @allure.description(\"Проверка граничных условий полей для ввода имени, фамилии и пароля\")\n    @pytest.mark.parametrize(\"first_name\",[UserLoginPage.generate_string(num_of_chars=1),\n                                           UserLoginPage.generate_string(num_of_chars=32, num=True, uppercase=True)])\n    @pytest.mark.parametrize(\"second_name\",[UserLoginPage.generate_string(num_of_chars=1),\n                                           UserLoginPage.generate_string(num_of_chars=32, num=True, uppercase=True)])\n    @pytest.mark.parametrize(\"password\",\n                             [UserLoginPage.generate_string(num_of_chars=4, num=True, uppercase=True, sym=True),\n                              UserLoginPage.generate_string(num_of_chars=20, num=True, uppercase=True, sym=True)])\n    def test_border_conditions(browser, base_url, first_name, second_name, password):\n        new_user = UserLoginPage(browser)\n        new_user.open_reg_account_page(base_url)\n        new_user.fill_in_with_letters(new_user.FIRST_NAME_FIELD, name=first_name)\n        new_user.fill_in_with_letters(new_user.LAST_NAME_FIELD, name=second_name)\n        new_user.enter_email()\n        new_user.enter_telephone()\n        passwd = new_user.enter_password(pwd=password)\n        new_user.fill_in_with_letters(new_user.PASSWORD_CONFIRM_FIELD, name=passwd)\n        new_user.find_web_element(new_user.PRIVACY_POLICY_CHECKBOX).click()\n        new_user.wait_and_click(new_user.SUBMIT_CONTINUE_BUTTON)\n        if new_user.view_success_notification() != \"Your Account Has Been Created!\":\n            browser.save_screenshot(new_user.screenshot_name)\n            new_user.logger.error(\"Account creation message was not displayed \")\n>           raise AssertionError(\"No message\")\nE           AssertionError: No message\n\ntest_reg_account.py:66: AssertionError"}, "description": "Проверка граничных условий полей для ввода имени, фамилии и пароля", "attachments": [{"name": "log", "source": "ecf4c048-5e44-4134-b30c-5e11154df1ce-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "password", "value": "'e$kWq2i+a&cY9bJtB4OI'"}, {"name": "second_name", "value": "'t'"}, {"name": "first_name", "value": "'e39akyNUA4ZfnglSV5sX2bFYQrhTqjIi'"}], "start": 1640005761126, "stop": 1640005764937, "uuid": "a1a2f15a-1255-4786-a4b2-6ed3ca162ac0", "historyId": "fcde07dc72498f78e4fd1e3a23094d08", "testCaseId": "bb2220456bb940b18fcd35ce6dab0375", "fullName": "test_reg_account#test_border_conditions", "labels": [{"name": "suite", "value": "test_reg_account"}, {"name": "host", "value": "sis002422"}, {"name": "thread", "value": "4616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_reg_account"}]}